src/cms-content/learn/learn-faq.json

```diff
@@ -17,12 +17,12 @@
         {
           "question": "When can children get the COVID vaccine?",
           "questionId": "when-can-children-get-covid-vaccine",
-          "answer": "The [FDA has officially authorized use](https://www.fda.gov/news-events/press-announcements/fda-authorizes-pfizer-biontech-covid-19-vaccine-emergency-use-children-5-through-11-years-age) of the Pfizer vaccine for everyone 5 years and older. This does not include booster shots, which are currently [approved for those 12 and older](https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-receive-us-fda-emergency-use-0#:~:text=The%20Pfizer%2DBioNTech%20COVID%2D19%20Vaccine%20has%20received%20EUA%20from,have%20certain%20kinds%20of%20immunocompromise). Moderna’s vaccine trials in children aged 6 to 11 yielded positive results in terms of safety and efficacy, but it is still pending FDA approval for this age group.\n\nTrials with children under the age of 5 are already underway for both Pfizer and Moderna vaccines. The CDC updates their [recommendations](https://www.cdc.gov/coronavirus/2019-ncov/vaccines/recommendations/children-teens.html) as new age groups become eligible."
+          "answer": "The [FDA has officially authorized use](https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-moderna-and-pfizer-biontech-covid-19-vaccines-children) of the Pfizer vaccine for everyone 6 months and older. This does not include booster shots, which are currently [approved for those 5 and older](https://www.cdc.gov/coronavirus/2019-ncov/vaccines/booster-shot.html#when-you-can-get-booster). \n\nThe CDC updates their [recommendations](https://www.cdc.gov/coronavirus/2019-ncov/vaccines/recommendations/children-teens.html) as new age groups become eligible."
```

Comment:
Isn't this inconsistent with the paragraph below? Here:

> The FDA has officially authorized use of the Pfizer vaccine for everyone 6 months and older.

The answer below for _Which COVID vaccine should you get?_:

> The FDA has officially authorized use of the Pfizer and Moderna vaccines for children 6 months and older.

cc @mikaylaobryan

Reply:
Should be: "The FDA has officially authorized use of the Pfizer and Moderna vaccines for children 6 months and older."